### ISSUE TITLE
Players can break item frames in war events even with allow_block_griefing set to false

### ISSUE THREAD

**Issue by ipozow** [TownyAdvanced/Towny#4447]
### What steps will reproduce the problem?
1. /ta toggle war
2. Find a protected item frame
3. Break it

### What is the expected output?
1. War starts
2. Find a protected item frame
3. Item frame should be not destroyed

### What is your Towny version?
0.96.3.0

### What is your spigot/craftbukkit version?
Do not say "latest".
git-Paper-216 1.16.3

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log :
2. Your Towny config.yml (if using MySQL please remove password): https://pastebin.com/GFeSbkBF
3. Your townyperms.yml: https://pastebin.com/MSKJduuL
4. Your log's error : 

(optional) If this is to do with permissions, 
5. Your permissions file :

**Comment by LlmDl** [comment]
You config says you're not on 0.96.3.0

Show me the full startup.

**Comment by ipozow** [comment]
True. I am sorry, I confused servers. This one was actually 0.96.2.19 at the moment of war. 

```
[10:17:34] [Server thread/INFO]: [Towny] Enabling Towny v0.96.2.19
[10:17:34] [Server thread/INFO]: ====================      Towny      ========================
[10:17:34] [Server thread/INFO]: [Towny] Lang: english.yml v0.86 loaded.
[10:17:34] [Server thread/INFO]: [Towny] Database: [Load] mysql [Save] mysql
[10:17:36] [Server thread/INFO]: [Towny] Database: Loaded in 1288ms.
[10:17:36] [Server thread/INFO]: [Towny] Database: 100% of residents have stored UUIDs.
[10:17:36] [Server thread/INFO]: [Towny] Searching for third-party plugins...
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: [Towny] Successfully cleaned backups.
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: Performing backup
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: ***** Warning *****
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: ***** Only Snapshots and Regen files will be backed up
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: ***** Make sure you schedule a backup in MySQL too!!!
[10:17:36] [Server thread/INFO]: [PlaceholderAPI] Successfully registered expansion: townyadvanced
[10:17:36] [Server thread/INFO]: [Towny] Plugins found: 
[10:17:36] [Server thread/INFO]:   Permissions: TownyPerms, Vault 1.6.11 via Vault
[10:17:36] [Server thread/INFO]:   Economy: CMIEconomy via Vault
[10:17:36] [Server thread/INFO]:   Add-ons: TownyChat v0.78, PlaceholderAPI v2.10.9
[10:17:36] [Server thread/INFO]: [Towny] Time until a New Day: 1 hours, 42 minutes, 24 seconds
[10:17:36] [ForkJoinPool.commonPool-worker-5/INFO]: [Towny] Towny flatfiles and settings successfully backed up.
[10:17:36] [Server thread/INFO]: =============================================================
[10:17:36] [Server thread/INFO]: [Towny] Version: 0.96.2.19 - Plugin Enabled
[10:17:36] [Server thread/INFO]: =============================================================
```

**Comment by LlmDl** [comment]
Ok, well the good news is that item frames ought to be fully protected in the current pre-releases because I've changed how all of that stuff works. Test it out with 0.96.3.6 (or newer) and if there's still an issue please re-file the ticket. I'm currently revamping Event War in a big way so I'd like to know if there's issues.

**Comment by ipozow** [comment]
but that pre-release already had the item frames fixes, I tried again with 0.96.3.6 and players can still break item frames in war

**Comment by ipozow** [comment]
it's fixed in Towny-0.96.3.7.jar
